examples/webgl_loader_ifc.html

```diff
@@ -61,10 +61,10 @@
 
 				//Setup IFC Loader
 				const ifcLoader = new IFCLoader();
-				ifcLoader.setWasmPath( 'jsm/loaders/ifc/' );
+				ifcLoader.ifcManager.setWasmPath( 'jsm/loaders/ifc/' );
 				ifcLoader.load( 'models/ifc/rac_advanced_sample_project.ifc', function ( model ) {
 
-					scene.add( model );
+					scene.add( model.mesh );
```

Comment:
`IFCLoader` returns a single `Mesh` now then?

Reply:
Indeed. In the beginning we were generating an object3d containing one mesh per item in the IFC file, but we found that this was already too laggy in medium sized files. The solution we came up with was generating only one single mesh (with one group per material), which [dramatically increased the performance](https://twitter.com/agviegasBIM/status/1390053366903021572). That's the reason why we provide an API; to be able to access the individual elements, highlight them, get their properties, etc. That way, users can both have big models at 60 fps and still be able to work with individual items, as if they were separate meshes.

Reply:
I see...

How about making `IFCLoader` return a `IFCModel` that extends `Mesh` then?

Then we can just do:

```js
scene.add( model );
```

Reply:
We don't mind, but we would like a firm decision before changing the API and [documentation](https://ifcjs.github.io/info/docs/Guide/web-ifc-three/api#ifcloader) again 😅 We have already made the API accessible from the IfcModel instances, not only from the `IfcLoader`. If we make the IfcLoader return an `IfcMesh`, should we provide each `IfcMesh` with all the methods of the API? For example: `IfcMesh.getExpressID()` .

Reply:
Yes, sorry...

When I suggested returning a `Group` instead of a `Scene` I was not aware that the the `Scene` had only one child.

We could follow `GLTFLoader` approach which returns an object:

```js
loader.load( 'foo.ifc', function ( ifc /* { manager : IFCManager, mesh : Mesh } */ ) {

    const mesh = ifc.mesh;
    scene.add( ifc.mesh );

    const manager = ifc.manager;
    const id = manager.getExpressId( mesh.geometry );

} );
```

This approach would also be backwards compatible.

Reply:
No worries! Sounds good, we'll do that 👌

Reply:
Does this look correct?
```ts
export class IFCModel extends Mesh {
    modelID = modelIdCounter++
    constructor(public mesh: Mesh, public ifc: IFCManager) {
        super();
    }
}
```

Reply:
This seems safer?

```js
export class IFCModel extends Mesh {
    modelID = modelIdCounter++;
    mesh = this;
    manager = null;

    setIFCManager( manager ) {
        this.manager = manager;
    }
}
```